src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -127,6 +154,15 @@ public string SasKey
             set => this.sasKey = value.Trim();
         }
 
+        /// <summary>
+        /// Get the transport type from the connection string
+        /// </summary>
+        public TransportType TransportType
+        {
+            get => this.transportType;
+            set => this.transportType = value;
+        }
```

Comment:
There's no harm with this code, but this could be just be `{ get; set; }`, and we can get rid of declaring the variable `transportType`